src/screens/GroupInfo/elements/ModalAction.js

```diff
@@ -62,7 +62,7 @@ const styles = StyleSheet.create({
   disabledTextButton: {
     fontSize: 16,
     fontWeight: 'bold',
-    color: COLORS.gray1
+    color: COLORS.lightgrey
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.gray1` to `COLORS.lightgrey` seems fine as long as the new color `lightgrey` is defined in the `COLORS` object and it fits well with the overall theme of the application. Please ensure that this color change doesn't affect the readability or user experience.
```diff
-    color: COLORS.gray1
+    color: COLORS.lightgrey
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->